src/Dom/Browser.Dom.fs

```diff
@@ -360,14 +360,14 @@ type [<AllowNullLiteral>] Document =
     abstract writeln: [<ParamArray>] content: string[] -> unit
 
 type [<AllowNullLiteral>] DocumentType =
+    inherit Node
```

Comment:
I added this while doing a brief review of the [DocumentType](https://developer.mozilla.org/en-US/docs/Web/API/DocumentType) API on MDN.